### ISSUE TITLE
Operator tutorial outdated

### ISSUE THREAD

**Issue by agardnerIT** [open-feature/killercoda#6]
The [killercoda Operator tutorial](https://killercoda.com/open-feature/scenario/openfeature-operator-demo) is outdated and broken. Thanks @salaboy who made me realise this.

1. Helm chart is now `open-feature/open-feature-operator` not [open-feature/ofo](https://github.com/open-feature/killercoda/blob/505c1da473e85c33934342be654d91d325a70e3a/openfeature-operator-demo/assets/scripts/intro_foreground.sh#L19)
2. Need `kubectl create namespace open-feature-demo`
3. kubectl apply end-to-end.yaml needs a namespace added
4. Need to create a standlone featureflagconfiguration for two reasons: 1) It makes editing more logical. 2) kubectl applying end-to-end fails as it contains the service which is already bound to port 30000.
5. Move the port-forwarding to a `nohup` operation in the creation script

In short, this issue will fix the [OpenFeature operator Killercoda tutorial](https://killercoda.com/open-feature/scenario/openfeature-operator-demo) and bring it up to date.

**Comment by agardnerIT** [comment]
Re-opening until #10 is merged.

**Comment by agardnerIT** [comment]
This is now resolved.
